charts/seed-controlplane/charts/kube-apiserver/templates/kube-apiserver.yaml

```diff
@@ -61,6 +61,7 @@ spec:
         - --client-ca-file=/srv/kubernetes/ca/ca.crt
         - --enable-aggregator-routing=true
         - --enable-bootstrap-token-auth=true
+        - --http2-max-streams-per-connection=1000
```

Comment:
This flag is only supported from `>= 1.10.x`.
https://v1-9.docs.kubernetes.io/docs/reference/generated/kube-apiserver/

Reply:
The documentation seem out of date.
https://github.com/kubernetes/kubernetes/pull/60685
https://github.com/kubernetes/kubernetes/pull/60802

I also chcked with hyperkube v1.9.8 help section from image `k8s.gcr.io/hyperkube:v1.9.8`. I could see the flag `http2-max-streams-per-connection`.

Reply:
Done. Thanks @mvladev